<launch>
   <!-- <arg name="cloud_topic" default="/xtion/depth/points" /> -->

  <!-- Start Perception -->
  <!-- <node name="basic_grasping_perception" pkg="simple_grasping" type="basic_grasping_perception" >
    <rosparam command="load" file="$(find phoebe_sim)/config/simple_grasping.yaml" />
	<remap from="/head_camera/depth_registered/points" to="$(arg cloud_topic)"/>
  </node> -->
  <!-- Start Navigation Stack -->
  <!-- <include file="$(find phoebe_navigation)/launch/phoebe_navigation.launch"/> -->
  
  <!-- Start Perception Server. It provides the location of handles -->
  <node name="perception" pkg="phoebe_perception" type="perception_node" output="screen" /> 

  <!-- Drive to the table, pick stuff up -->
  <node name="demo" pkg="phoebe_sim" type="demo.py" output="screen" />   <!-- demo  pick_and_place -->

</launch>
